core/backend/src/TextAnnotationLayout.ts

```diff
@@ -244,6 +244,17 @@ function split(source: string): Segment[] {
   return segments;
 }
 
+/**
+ * TypeScript only provides type declarations for Intl.Segmenter if targeting ES2022+.
+ * Use this type as a stand-in until ES2022+ is supported.
+ * @internal
```

Comment:
Since this interface isn't exported, `@internal` isn't necessary?

Reply:
Thanks Ben, this interface has been removed

Reply:
Thanks!